### PR TITLE
SOL-11110 Use Go 1.20

### PR DESCRIPTION
**Pull request by andreiovidiu**
Problem:
BNYM reported 3 CVEs related to monstache's dependency to go

Solution:
Bump to version 1.20.4+

Steps:
- Installed Go 1.20.5 locally
- Changed go.mod: `go 1.13` -> `go 1.20`
- go install; go build

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -18,4 +14,26 @@ require (
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 )
 
-go 1.13
+require (
+	github.com/golang/snappy v0.0.4 // indirect
+	github.com/jmespath/go-jmespath v0.4.0 // indirect
```
Comment (mix4242) [inline go.mod:19]: What’s the reason for so many new lines here? Or is it automatic or smt?

Comment (andreiovidiu) [reply go.mod:19]: yes, they were automatically generated

Comment (mix4242) [reply go.mod:19]: Calm

File: go.mod [hunk 2]
```diff
@@ -18,4 +14,26 @@ require (
...
+	github.com/xdg-go/stringprep v1.0.3 // indirect
+	github.com/youmark/pkcs8 v0.0.0-20201027041543-1326539a0a0a // indirect
+	golang.org/x/crypto v0.0.0-20220622213112-05595931fe9d // indirect
+	golang.org/x/sync v0.0.0-20220722155255-886fb9371eb4 // indirect
+	golang.org/x/sys v0.1.0 // indirect
+	golang.org/x/text v0.4.0 // indirect
+	gopkg.in/sourcemap.v1 v1.0.5 // indirect
+)
+
+go 1.20
```
Comment (mix4242) [inline go.mod:39]: How does it work that here it's just `1.20` yet we go to `1.20.4+`? Does it automatically pick latest `1.20` installed on system? What if someone didn't have `1.20.4+` installed? Can we lock patch ver too?

**Review by mix4242 — CHANGES_REQUESTED** [review]
In description could u drop few lines how u upgraded the version, like what command u used etc :)

**Review by mix4242 — APPROVED** [review]

**Review by mix4242 — CHANGES_REQUESTED** [review]
Acc could we bump monstache to `6.7.10.1` (search & repalce all occurances of `6.7.10.0` :D)

**Review by mix4242 — APPROVED** [review]

**Pull request merged**